### PR TITLE
move api groups to v1beta1

### PR DESCRIPTION
**Pull request by mhenriks**
**What this PR does / why we need it**:

Updates api groups to v1beta1.  v1alpha1 is still supported if resources created with yaml/old clients.  No webhook conversion necessary ATM.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

obsoletes #1196

**Release note**:

```release-note
API Groups updated to v1beta1
```

### CODE REVIEW

**Comment by awels** [comment]
Ugh, looks like need some rebasing and there are some compile failures in the datavolume_test.go. But it looks good otherwise, and I will try to get it merged asap.

File: cluster-up/hack/common.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@ fi
 
 
 KUBEVIRTCI_CLUSTER_PATH=${KUBEVIRTCI_CLUSTER_PATH:-${KUBEVIRTCI_PATH}/cluster}
-KUBEVIRT_PROVIDER=${KUBEVIRT_PROVIDER:-k8s-1.18}
+KUBEVIRT_PROVIDER=${KUBEVIRT_PROVIDER:-k8s-1.15}
```
Comment (awels) [inline cluster-up/hack/common.sh:20]: This should be 1.18

Comment (mhenriks) [reply cluster-up/hack/common.sh:20]: `make generate` produced this so it looks like kubevirtci should be updated

Comment (mhenriks) [reply cluster-up/hack/common.sh:20]: oh wait

Comment (mhenriks) [reply cluster-up/hack/common.sh:20]: Or `make generate` updates this

Comment (awels) [reply cluster-up/hack/common.sh:20]: Hah should teach me to review late at night, didn't even notice it was in cluster-up. Yeah this is all generated so out of your control.

**Pull request merged**